packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.Props.ts

```diff
@@ -59,4 +59,27 @@ export interface IBaseAutoFillProps extends React.HTMLAttributes<HTMLInputElemen
    */
   enableAutoFillOnKeyPress?: KeyCodes[];
 
+  /**
+   * the default value to be visible
+   */
+  defaultVisibleValue?: string;
+
+  /**
+   * Handler for checking and updating the value if needed
+   *  in componentWillReceiveProps
+   *
+   * @param {IBaseAutoFillProps} defaultVisibleValue - the defaultVisibleValue that got passed
+   *  in to the auto fill's componentWillReceiveProps
+   * @returns {string} - the updated value to set, if needed
+   */
+  updateValueInWillReceiceProps?: () => string;
```

Comment:
nit: there is a typo in here. Is it common to add the lifecycle function where these prop functions are used in Fabric? It looks to me as if the name itself is leaking already implementation details of the component.